### ISSUE TITLE
About downstream task and DropPath

### ISSUE THREAD

**Issue by Mayner0220** [naver-ai/rdnet#5]
Thank you for your great ideas and work.

I have currently re-implemented RDNet Backbone with Tensorflow, and I am trying to apply it to Human Pose Estimation(HPE) as a downstream task and I am asking you because I have the following questions:

1. From what I understand after reading RDNet paper, RDNet backbone uses Stochastic Depth(DropPath), but RDNet's [`DenseBlock`](https://github.com/naver-ai/rdnet/blob/78360d02058fd1e37c992c45c34e99a3f62f2bd8/segmentation/backbone/rdnet.py#L95-L133) class only declares `DropPath` and doesn't seem to use it. Is this the part that was intended? If yes, please explain why you didn't use it.
2. As a downstream task, I am trying to train a model by applying RDNet to the HPE task, but I have confirmed that the model is not trained well. I have tried several times by modifying hyperparameters such as batch size, LR, etc., but the model is not trained well. As a curiosity about this, could the training be sensitive or not trained well when applying RDNet Backbone to the downstream task?
The conditions used in training are as follows:
    - Backbone: RDNet-Tiny (pretrained: X)
    - LR Scheduler: MultiStepLR
    - LR: 0.001 (also tried lower LR)
    - batch size: 64 (also tried 16, 32)
    - optimizer: Adam
---
<details>
<summary>한국어 버전(Korean Translation)</summary>
RDNet이라는 좋은 아이디어와 결과물을 공유 해주셔서 감사합니다.

현재 저는 RDNet Backbone을 Tensorflow로 재 구현하여, downstream task로 Human Pose Estimation(HPE)에 적용하고자 하는 상황 중에 궁금한 점이 생겨 아래의 질문들을 문의드립니다:

1. RDNet 논문을 이해하기로 RDNet Backbone은 Stochastic Depth(DropPath)를 사용하는 것으로 알고 있습니다. 하지만 RDNet의 [`DenseBlock`](https://github.com/naver-ai/rdnet/blob/78360d02058fd1e37c992c45c34e99a3f62f2bd8/segmentation/backbone/rdnet.py#L95-L133) 클래스에서 `DropPath`를 선언만 하고 사용하지 않는 것으로 보입니다. 이 부분이 의도된 것이 맞는 문의드립니다. 만약, 의도된 사항이 맞다면 그 이유도 설명 부탁드립니다.
2. downstream task로 RDNet Backbone을 HPE task에 적용하고자 시도한 결과, 모델의 학습이 잘 이루어지지 않음을 확인했습니다. batch size나 LR과 같은 hyperparamter도 수정하여 여러 번 시도 해봤지만 이 역시 학습이 잘 되지 않았습니다. 이에 대한 궁금증으로 RDNet Backbone을 downstream task에 적용할 경우, 학습이 민감해지거나(hyperparameter 등으로 인해), 학습이 잘 안될 수도 있을지 궁금합니다:
학습 때 적용한 조건들은 다음과 같습니다.
    - Backbone: RDNet-Tiny (pretrained: X)
    - LR Scheduler: MultiStepLR
    - LR: 0.001 (also tried lower LR)
    - batch size: 64 (also tried 16, 32)
    - optimizer: Adam
</details>

**Comment by dhkim0225** [comment]
Sincerely appreciate your interest in our work!

1. During the process of creating the release-specific code, there was an instance of code omission. I apologize for this oversight. I will ensure that the necessary corrections are implemented and provide you with an update once completed. (cc @ogencoglu )

2. All the tables and figures reported in the paper (for downstream tasks) are based on experiments conducted using IN-1k pretrained weights. For RDNet, I empirically observe that its initial convergence is slower compared to other networks. This can be interpreted as a difficulty in effectively training the network in its early stages. When the pretrained argument is set to False and the network is trained on downstream tasks, this issue can persist. I believe that considering pretraining is important for RDNet, especially when the downstream data is limited and the training epochs are short.

**Comment by dhkim0225** [comment]
The bug has been fixed. Please check it on release 0.1.2 ! 

Additionally, I plan to integrate RDNet into the timm package. 
timm issue : https://github.com/huggingface/pytorch-image-models/issues/2241

I will re-train RDNet during this process and re-validate the code for any potential bugs.

Stay tuned for updates!

**Comment by Mayner0220** [comment]
Thank you for checking and fixing the bug quickly!

I will close this issue.
